ui/theme/theme.go

```diff
@@ -160,6 +170,26 @@ func (m *MyTheme) ListThemeFiles() map[string]string {
 	return result
 }
 
+func (m *MyTheme) createThemeResources() error {
+	err := configdir.MakePath(m.themeFileDir)
```

Comment:
Here and elsewhere, we want to log out errors (I'll have to improve the logging later but for now it just logs to stdout), and use the compact if statement:

```go
if err := doSomething(); err != nil {
    log.Println("error doing something: %s", err.Error())
    return err
}